src/docs/admin/ProActiveAdminGuide.adoc

```diff
@@ -65,6 +65,11 @@ The URLs of the Scheduler, Resource Manager, REST API and Web Interfaces are dis
 
 Your ProActive Scheduler is ready to execute <<_glossary_job,*Jobs*>>!
 
+=== Minimum Requirements
+
+The minimum requirements can be found in the Data Sheet. Please find an up to date Data Sheet in the in the link:https://www.activeeon.com/about/resources[resources].
+
+The `proactive-server` script allocates 4GB heap space. One can change the allocated heap by changing the corresponding options in the `JAVA_OPTS` variable in the `proactive-server` script.
```

Comment:
Maybe we can add the name of the option to change, or a usage example.

Reply:
I think it would make sense to add some information about JVM properties in the ProActive Scheduler configuration chapter, example

Various command-line tools shipped with ProActive (bin/proactive-server, bin/proactive-node) start a Java Virtual Machine. 
The parameters of the JVM can be modified by editing in the corresponding script the following line:
Linux:

Windows:

For example, to define a maximum heap capacity on the JVM, set:

Reply:
Very good points. I added the JVM maximum heap size configuration with an example to an Java Virtual Machine Configuration section.